### ISSUE TITLE
Augustus step not working, empty files

### ISSUE THREAD

**Issue by JulieRamMar** [Gaius-Augustus/BRAKER#720]
Hello, I'm using BRAKER3 to annotate the genomes of several a fungi, I have several species. The problem is that for some of them the Augustus files are not generated or are empty files, and I don't know why. 
For example, for a given species I have 8 assemblies, and I sequenced the RNA of one of those 8 individuals. Then I'm using this RNAseq data to annotate all 8 of them. For some of the assemblies all works normally, but for some of them Ausgustus doesn't seem to be working because I get empy files.
Why can that happen? the assemblies are very similar cuz they are from the same species, their quality is very similar as well, and the RNA data I use to annotate them is the same.

Please help

Thank you

**Comment by KatharinaHoff** [comment]
This is very weird. Are you running jobs on parallel?

**Comment by JulieRamMar** [comment]
I use a pipeline in which everything starts more or less at the same time, I mean, it creates and lunches jobs for each sample individually.

**Comment by KatharinaHoff** [comment]
This might be a problem. Are you specifying species names to the jobs (--species=someNameForEachJob)?

**Comment by JulieRamMar** [comment]
yes, each job has a different name, e.g.: braker.smp=Pteres_Assembly12
